app/frontend/components/pages/login/generateMnemonicDialog.js

```diff
@@ -18,7 +19,7 @@ class GenerateMnemonicDialogClass {
         'The new wallet is created together with a mnemonic phrase. Write the mnemonic phrase down, you will need it to access your wallet. ',
         h('strong', undefined, 'Don’t copy it to your clipboard or save it anywhere online.')
       ),
-      h('div', {class: 'modal-mnemonic one-click-select'}, mnemonic),
+      h('div', {class: 'modal-mnemonic one-click-select'}, generateMnemonic()),
```

Comment:
this isn't a good idea since re-rendering the component would trigger mnemonic regeneration. I would rather keep the previous logic but wipe the generated mnemonic from the state once the user closes the generate mnemonic dialog.

Reply:
Ok , but it has to be different state parameter to avoid mnemonic field to be filled out on background and than cleared when user closes the window. It looks glitchy. For example generatedMnemonic.